server/backend/users/views.py

```
@@ -130,6 +127,60 @@ def post(self, request):
 
 
 
+# ----------------------------- RESET PASSWORD --------------------------------
+class RequestPasswordResetView(APIView):
+    permission_classes = [AllowAny]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)